lib/l10n/app_en.arb

```diff
@@ -418,9 +428,17 @@
     "l_authenticator_get_started": "Get started with OTP accounts",
     "l_no_accounts_desc": "Add accounts to your YubiKey from any service provider supporting OATH TOTP/HOTP",
     "s_add_account": "Add account",
+    "p_add_account_desc": "Scan QR code (recommended) or enter the credential details manually.",
     "s_add_accounts": "Add account(s)",
     "p_add_description": "To scan a QR code, make sure the full code is visible on screen and press the button below. You can also drag a saved image from a folder onto this dialog. If you have the account credential details in writing, use the manual entry instead.",
     "l_drop_qr_description": "Drop QR code to add account(s)",
+    "p_add_account_three_ways": "There's 3 ways of adding accounts.",
+    "s_scanning": "Scanning (recommended)",
+    "p_scanning_desc": "Before scanning a QR code, make sure the full code is visible on screen.",
+    "s_drag_and_drop": "Drag and drop",
+    "p_drag_and_drop_desc": "An image containing a QR code may be dropped anywhere in the Accounts application.",
```

Comment:
We use "application" to refer to actual code running on the YubiKey. I think we need something else to refer to the "view" in the app.

Reply:
Would "view" work?

Reply:
Changed it to "view". Let me know what you think.